packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -114,9 +114,10 @@ describe('Dialog', () => {
 
       let callbackCalled = false;
       const dialogInfo: DialogInfo = {};
-      dialog.open(dialogInfo, (err, result) => {
-        expect(err).toBeNull();
-        expect(result).toBe('someResult');
+      dialog.open(dialogInfo, resultObj => {
```

Comment:
Do we have unit tests for any of the new open functionality as well? (Passing in messageForChildHandler, making sure the new return value does what we expect it to, etc.)

Reply:
I added some more unit tests. Please check them out.